tests/testthat/test-z-integration.R

```diff
@@ -685,21 +686,6 @@ test_that("can get report versions", {
 })
 
 
-test_that("Returns 404 if no report versions", {
```

Comment:
moved these from integration tests to endpoint tests (since former are much slower than the latter)